lib/MusicBrainz/Server/Data/Statistics.pm

```diff
@@ -398,6 +398,20 @@ my %stats = (
             };
         },
     },
+    'count.event.has_art' => {
+        DESC => 'Count of events with event art',
+        SQL => 'SELECT COUNT(distinct event) FROM event_art_archive.event_art',
+    },
+    'count.event.has_front_art' => {
+        DESC => 'Count of events with front event art',
+        SQL => 'SELECT COUNT(distinct event) FROM event_art_archive.event_art ea
+                  JOIN event_art_archive.event_art_type eat ON ea.id = eat.id
+                WHERE eat.type_id = 1',
```

Comment:
Is 1 poster, and if so, should this be called `has_poster_art`

Reply:
Should I also change the description to "Count of events with poster art"?

Reply:
Maybe? I think that's a lot more clear for humans.